omnij-jsonrpc/src/main/java/foundation/omni/rpc/OmniClient.java

```diff
@@ -311,6 +311,22 @@ public Sha256Hash omniSendAll(Address fromAddress, Address toAddress, Ecosystem
         return send("omni_sendall", Sha256Hash.class, fromAddress, toAddress, ecosystem);
     }
 
+    /**
+     * Creates and broadcasts a "send to many" transaction.
+     *
+     * @param fromAddress The address to spent from
+     * @param currency    The identifier of the token to distribute
+     * @param mapping     The receiving addresses and amounts
+     * @return The hash of the transaction
+     * @throws JsonRpcException JSON RPC error
+     * @throws IOException network error
+     * @since Omni Core 0.0.12
+     */
+    public Sha256Hash omniSendToMany(Address fromAddress, CurrencyID currency, List<Map<String, Object>> mapping)
```

Comment:
This API should use the `OmniOutput` type.

Reply:
Updated